expo-maps/android/src/main/java/expo/modules/maps/GoogleMapsView.kt

```diff
@@ -27,6 +27,10 @@ class GoogleMapsView(context: Context): LinearLayout(context), OnMapReadyCallbac
 
   override fun onMapReady(googleMap: GoogleMap) {
     this.googleMap = googleMap
+    // Possibly shift this logic to a dedicated method, when there is more dependencies
+    if(gestures != null) {
+      (gestures as GoogleMapsViewGestures).initMap(googleMap)
+    }
```

Comment:
As far as I understand this will be run only once (?) so we can just put constructor here and pass `googleMap` as its param. What do you think? Please verify if you understand this doc same way as I do.

CC: @p-karcz 

https://developers.google.com/android/reference/com/google/android/gms/maps/OnMapReadyCallback

```suggestion
    gestures = GoogleMapsGestures(googleMap)
```